rbx_binary/src/serializer/state.rs

```diff
@@ -1077,14 +1077,15 @@ impl<'dom, 'db, W: Write> SerializerState<'dom, 'db, W> {
                         for (i, rbx_value) in values {
                             if let Variant::PhysicalProperties(value) = rbx_value.as_ref() {
                                 if let PhysicalProperties::Custom(props) = value {
-                                    chunk.write_u8(1)?;
-                                    chunk.write_le_f32(props.density)?;
-                                    chunk.write_le_f32(props.friction)?;
-                                    chunk.write_le_f32(props.elasticity)?;
-                                    chunk.write_le_f32(props.friction_weight)?;
-                                    chunk.write_le_f32(props.elasticity_weight)?;
+                                    chunk.write_u8(0b11)?;
+                                    chunk.write_le_f32(props.density())?;
+                                    chunk.write_le_f32(props.friction())?;
+                                    chunk.write_le_f32(props.elasticity())?;
+                                    chunk.write_le_f32(props.friction_weight())?;
+                                    chunk.write_le_f32(props.elasticity_weight())?;
+                                    chunk.write_le_f32(props.acoustic_absorption())?;
                                 } else {
-                                    chunk.write_u8(0)?;
+                                    chunk.write_u8(0b10)?;
```

Comment:
Don't love how we'll end up modifying preexisting PhysicalProperties -- maybe we could write `0b00` if there are no custom properties, `0b01` if there are custom properties and acoustic absorption == 1, and `0b11` if there are custom properties and acoustic absorption ~= 1?